src/sources/youtube.js

```diff
@@ -723,23 +698,20 @@ function loadLyrics(decodedTrack, language) {
 
   return new Promise(async (resolve) => {
     if (!config.search.sources.youtube.bypassAgeRestriction)
-      _switchClient(decodedTrack.sourceName === 'ytmusic' ? 'ANDROID_MUSIC' : 'ANDROID')
-
-    const { body: video } = await makeRequest(`https://${_getBaseHostRequest(decodedTrack.sourceName)}/youtubei/v1/player`, {
-      headers: {
-        'User-Agent': ytContext.client.userAgent,
-        'X-GOOG-API-FORMAT-VERSION': '2',
-        ..._additionalHeaders
-      },
+      _switchClient(decodedTrack.sourceName === 'ytmusic' ? 'ANDROID_MUSIC' : 'IOS')
+    
+    const { body: video } = await makeRequest(`https://${_getBaseHostRequest(type)}/youtubei/v1/player`, {
       body: {
-        context: ytContext,
+        context: {
+          client: ytContext.client
+        },
         videoId: decodedTrack.identifier,
         contentCheckOk: true,
-        racyCheckOk: true,
-        params: 'CgIQBg'
+        racyCheckOk: true
       },
-      method: 'POST'
-    })
+      method: 'POST',
+      disableBodyCompression: true
+    });
```

Comment:
```suggestion
    })
```

Do not add semicolons